postgres-appliance/Dockerfile

```diff
@@ -70,7 +70,7 @@ COPY dependencies/debs dependencies/src /builddeps/
 ARG PGVERSION
 ARG TIMESCALEDB
 ARG TIMESCALEDB_LEGACY
-ARG TIMESCALEDB_APACHE_ONLY=true
+ARG TIMESCALEDB_APACHE_ONLY=false
```

Comment:
I am fine with the change of the default build argument in the Dockerfile, so everyone who builds the image on their own will automatically get Timescale with TSL.

But, we don't want to take responsibility for building and making publicly available the image with TSL license. Therefore please also update the [delivery.yaml](https://github.com/zalando/spilo/blob/master/delivery.yaml#L27) to
```
docker build --build-arg PGVERSION=$PGVERSION --build-arg TIMESCALEDB_APACHE_ONLY=true -t $IMAGE .
```

Also, it would be nice to reflect the change of the TIMESCALEDB_APACHE_ONLY default value in the [README](https://github.com/zalando/spilo/blob/master/README.rst#how-to-build-this-docker-image) file.

Reply:
Thank you for your reply and considering changing it.

Would you be open to two build tags, one with TSL and one without?

Reply:
I afraid we will not take a responsibility on distributing the software with non open-source (TSL) license.

Reply:
Ok. Thanks for meeting me half way, even so.